targets/xorg

```diff
@@ -205,8 +205,7 @@ else
 fi
 
 # Fix launching X11 from inside crosh (user doesn't own a TTY)
-sed -i 's/allowed_users=.*/allowed_users=anybody/' \
-    '/etc/X11/Xwrapper.config' || true
+echo 'allowed_users=anybody' > '/etc/X11/Xwrapper.config'
```

Comment:
Are we sure that `/etc/X11/Xwrapper.config` never contains anything else of interest?

If not, we might want to write a small awk script instead to modify/add the line as required.